### DISCUSSION TITLE
database not updating via table editor

### DISCUSSION THREAD

**Discussion by hexaaagon** [supabase/supabase#26690] [Questions]
when I update the database data through the Table Editor, it doesn't seem to update it.

using role "postgres role"

before (editing the database)
![image](https://github.com/supabase/supabase/assets/128217934/47398537-c154-4741-89f0-20b3b1a05f5f)

after (clicking the enter button & view the database again)
![image](https://github.com/supabase/supabase/assets/128217934/84a22f4a-05b5-47d8-9bcc-bb43cdb78643)

**Comment by silentworks** [comment]
Did you get any error? it's likely the SQL didn't execute.

**Reply by hexaaagon** [reply]
i'm not kinda sure but this is the error in the console:
![image](https://github.com/supabase/supabase/assets/128217934/bf2d943b-9b5e-4d87-865f-61c3ed5587df)

and this is in the network tab:
![image](https://github.com/supabase/supabase/assets/128217934/f0c2c293-195d-45bc-8518-14b301f03171)
